data/mods/Magiclysm/enchanted/enchanted_tokens.json

```diff
@@ -74,16 +74,91 @@
-    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a battle axe.",
+    "id": "mtoken_pike",
+    "name": "pike token",
+    "name_plural": "pike tokens",
+    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a pike.",
     "use_action": {
       "type": "transform",
-      "target": "battleaxe",
+      "target": "pike",
       "active": false,
       "moves": 200,
-      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine battle axe!"
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine pike!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_mace",
+    "name": "mace token",
+    "name_plural": "mace tokens",
+    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a mace.",
+    "use_action": {
+      "type": "transform",
+      "target": "mace",
+      "active": false,
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine mace!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_q_staff",
+    "name": "pike token",
+    "name_plural": "pike tokens",
+    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a quarterstaff.",
+    "use_action": {
+      "type": "transform",
+      "target": "q_staff",
+      "active": false,
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a pristine quarterstaff!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_hammer",
+    "name": "hammer token",
+    "name_plural": "hammer tokens",
+    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a hammer.",
+    "use_action": {
+      "type": "transform",
+      "target": "hammer",
+      "active": false,
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine hammer!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_screwdriver",
+    "name": "screwdriver token",
+    "name_plural": "screwdriver tokens",
+    "description": "A large silver coin that, when activated by uttering the word on the back, turns into the item pictured on the front, in this case a screwdriver.",
+    "use_action": {
+      "type": "transform",
+      "target": "screwdriver",
```

Comment:
This seems like a useless item due to how easy you can make a screwdriver, perhaps change to screwdriver set?

Reply:
Excellent idea, they were thrown in at the last minute, though it is intended for them to be somewhat common items to sprinkle around.